Exercises/loops/loops.js

```diff
@@ -16,3 +30,17 @@ const arrayToFilter = Array.from(new Array(1000), () => {
 
   return null;
 });
+
+const newArray = arrayToFilter.filter(item => Number.isInteger(item));
```

Comment:
the idea was not to use build-in filter, but if you are know how to do it with `for of` or `for (let i ....` I don't mind

Reply:
Implemented using `while`. I tried using `for of` but it worked correctly only for certain cases due to incorrect counter after deleting element.